docs/advanced-features/static-html-export.md

```diff
@@ -27,6 +27,7 @@ Update your `next.config.js` file to include `output: "export"` like the followi
  */
 const nextConfig = {
   output: 'export',
+  distDir: 'out',
```

Comment:
Should this be the default with `output: 'export'` to avoid differences from `next export`?

Reply:
It is the default. I just wanted to demonstrate how to change it.

Reply:
(see line `packages/next/src/build/index.ts:285` for the default)

Reply:
@ijjk Updated in [b5b9565](https://github.com/vercel/next.js/pull/47376/commits/b5b95659b508c52df632ce0cc0562f56ece6946d)